Example/Tests/EZYOverlayViewModelTests.swift

```diff
@@ -0,0 +1,169 @@
+//
```

Comment:
@shashankpali this is a new file which is added with Lint changes.

Reply:
Hey @max6363, it's because the **linting branch** is created from **UT's branch** which is not merged yet. That's why it showed up here.

Reply:
In PR I have mention that this [PR](https://github.com/shashankpali/EZYVideoPlayer/pull/2) needs to be merged first. Please have a look.